java/vector/src/test/java/org/apache/arrow/vector/ipc/TestArrowReaderWriter.java

```diff
@@ -70,6 +72,8 @@
 import org.junit.Before;
 import org.junit.Test;
 
+import com.fasterxml.jackson.databind.util.ByteBufferBackedInputStream;
```

Comment:
Should we depend on Jackson classes like this? I see a couple other uses of this in Arrow already, maybe (for a follow up) we should consider having our own implementation.

Reply:
(I suppose one of those uses is mine :joy:)

Reply:
Thanks for the good point. There is no need to depend on Jackson classes here.
I have revised the code to use ByteArrayInputStream instead. Please take another look. Thank you in advance.